addons/stock/models/stock_move_line.py

```diff
@@ -31,7 +31,7 @@ class StockMoveLine(models.Model):
     package_id = fields.Many2one('stock.quant.package', 'Source Package', ondelete='restrict')
     package_level_id = fields.Many2one('stock.package_level', 'Package Level')
     lot_id = fields.Many2one('stock.production.lot', 'Lot/Serial Number')
-    lot_name = fields.Char('Lot/Serial Number Name')
+    lot_name = fields.Text('Lot/Serial Number Name')
```

Comment:
maybe we should add lot_name2 as a text, keep lot_name as char and only display the text in the nosuggest view? :/